<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialNetworkMainWindow</class>
 <widget class="QMainWindow" name="SocialNetworkMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocialNetworkMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="loginPageLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-10</y>
      <width>561</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Login Page</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="usernameEntry">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Enter your Username Here:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="friendsList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Post3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>591</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Post4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Post2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>571</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>530</y>
      <width>581</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="addPost">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>600</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Add Post&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitNewPost">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>630</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="suggestedFriends">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>221</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back To Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddFriend">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Friend</string>
    </property>
   </widget>
   <widget class="QComboBox" name="privacyLevelBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>630</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <property name="Private" stdset="0">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="userBio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveUserBio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>82</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Bio</string>
    </property>
   </widget>
   <widget class="QComboBox" name="bioComboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="likePost2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>220</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Like Post 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="likePost3">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>310</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Like Post 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="likePost4">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>440</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Like Post 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="likePost5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>530</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Like Post 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="FriendsLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Friends</string>
    </property>
   </widget>
   <widget class="QLabel" name="SuggestedFriendsLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Suggested Friends</string>
    </property>
   </widget>
   <widget class="QPushButton" name="likePost1">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>130</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Like Post 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Post1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>599</width>
      <height>79</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToLoginButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back to Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
